dartagnan/src/main/java/com/dat3m/dartagnan/program/event/Tag.java

```diff
@@ -1,39 +1,36 @@
 package com.dat3m.dartagnan.program.event;
 
+/*
+    Tags can be attached to any event.
+    There are two types of tags:
+        (1) Tags referencable from CAT
+        (2) Tags used only internally (prefixed with a double underscore "__");
+ */
 public final class Tag {
     private Tag() { }
 
-    public static final String ANY              = "_";
+    public static final String VISIBLE          = "_";
     public static final String INIT             = "IW";
     public static final String READ             = "R";
     public static final String WRITE            = "W";
     public static final String MEMORY           = "M";
     public static final String FENCE            = "F";
+    public static final String STRONG           = "STRONG"; // TODO: Maybe move to C11 or IMM?
```

Comment:
Shouldn't it be `__STRONG`? If so, do we need this now that you added the `s.isStrong` field?

Reply:
I initially removed it, but `IMM` actually refers to this Tag, so we need it.
I would still keep the field instead of checkin for `e.is(Tag.STRONG)` in the code. One could argue that the Tag should only be generated when C11 (IMM) is the target and otherwise serves no purpose, while the field is always meaningful.